aws.go

```diff
@@ -73,15 +112,15 @@ func scrapeMetric(svc *cloudwatch.CloudWatch, metric Metric, instance string, co
 
 	params.Dimensions = append(params.Dimensions, &cloudwatch.Dimension{
 		Name:  aws.String("DBInstanceIdentifier"),
-		Value: aws.String(instance),
+		Value: aws.String(s.instance.Instance),
 	})
 
 	// Call CloudWatch to gather the datapoints
-	resp, err := svc.GetMetricStatistics(params)
-	collector.TotalRequests.Inc()
+	resp, err := s.svc.GetMetricStatistics(params)
+	s.collector.TotalRequests.Inc()
```

Comment:
I think collector can do it itself, no need to go from the scraper to collector.